test/source/mock/wkd/wkd-endpoints.ts

```diff
@@ -1,182 +1,61 @@
-OA/UwngEGBYIAAkFAmMEsqQCGwwAIQkQ+1x3UHaDHFEWIQQvI5Sm4OisrFQg
-XUP7XHdQdoMcUf0AAPwNsy3Z59HzQtZSx7pmP0vBbPMuWrFyg3Py8dB6A4lz
-CAD/VjKYjwJ4MYpcKZ7G3qYvrb3l7m2NTJLAi1yVTm1e5wU=
-=Dq31
------END PGP PRIVATE KEY BLOCK-----`;
-*/
+import { HttpClientErr, Status } from '../lib/api.js';
+import { isGet } from '../lib/mock-util.js';
+import { IncomingMessage } from 'http';
 
 // todo - add a not found test with: throw new HttpClientErr('Pubkey not found', 404);
 
-export const mockWkdEndpoints: HandlersDefinition = {
-  '/.well-known/openpgpkey/hu/st5or5guodbnsiqbzp6i34xw59h1sgmw?l=wkd': async () => {
-    // direct for wkd@google.mock.localhost:8001
-    const pub = await KeyUtil.asPublicKey(await KeyUtil.parse(testConstants.wkdAtgooglemockflowcryptlocalcom8001Private));
-    return Buffer.from((await PgpArmor.dearmor(KeyUtil.armor(pub))).data);
-  },
-  '/.well-known/openpgpkey/hu/n4qtbfz8ussx74ofsd1cbowqnaeoig3f?l=only.on.wkd': async () => {
-    return Buffer.from((await PgpArmor.dearmor(onlyOnWkdPub)).data); // for only.on.wkd@signing.test
-  },
-  '/.well-known/openpgpkey/hu/ihyath4noz8dsckzjbuyqnh4kbup6h4i?l=john.doe': async () => {
-    return Buffer.from((await PgpArmor.dearmor(johnDoe1)).data); // direct for john.doe@localhost
-  },
-  '/.well-known/openpgpkey/hu/ihyath4noz8dsckzjbuyqnh4kbup6h4i?l=John.Doe': async () => {
-    return Buffer.from((await PgpArmor.dearmor(johnDoe1)).data); // direct for John.Doe@localhost
-  },
-  '/.well-known/openpgpkey/hu/cb53pfqmbzc8mm3ecbjxyen65fdxos56?l=jack.advanced': async () => {
-    return Buffer.from((await PgpArmor.dearmor(jackAdvanced)).data); // direct for jack.advanced@localhost
-  },
-  '/.well-known/openpgpkey/localhost/hu/ihyath4noz8dsckzjbuyqnh4kbup6h4i?l=john.doe': async () => {
-    return Buffer.from((await PgpArmor.dearmor(johnDoe)).data); // advanced for john.doe@localhost
-  },
-  '/.well-known/openpgpkey/localhost/hu/ihyath4noz8dsckzjbuyqnh4kbup6h4i?l=John.Doe': async () => {
-    return Buffer.from((await PgpArmor.dearmor(johnDoe)).data); // advanced for John.Doe@localhost
-  },
-  '/.well-known/openpgpkey/localhost/hu/pob4adi8roqdsmtmxikx68pi6ij35oca?l=incorrect': async () => {
-    return Buffer.from((await PgpArmor.dearmor(alice)).data); // advanced for incorrect@localhost
-  },
-  '/.well-known/openpgpkey/localhost/hu/66iu18j7mk6hod4wqzf6qd37u6wejx4y?l=some.revoked': async () => {
-    return Buffer.from([
-      ...(await PgpArmor.dearmor(testConstants.somerevokedRevoked1)).data,
-      ...(await PgpArmor.dearmor(testConstants.somerevokedValid)).data,
-      ...(await PgpArmor.dearmor(testConstants.somerevokedRevoked2)).data,
-    ]);
-  },
-  '/.well-known/openpgpkey/localhost/policy': async () => {
-    return ''; // allow advanced for localhost
-  },
-  '/.well-known/openpgpkey/policy': async () => {
-    return ''; // allow direct for all
-  },
+interface WKDLookUpResult {
+  pubkeys?: string[];
+  returnError?: HttpClientErr;
+}
+
+export interface WKDConfig {
```

Comment:
let's call it `WkdConfig`, to be consistent with other names (like `FesConfig`)

Reply:
Sounds good!